zappnode-front/src/landing/components/features/features.js

```diff
@@ -8,7 +8,7 @@ import Button from '../button/button';
 const Features = () => {
     return(
         <section id="features" className="flex justify-center features flex-col py-16 mt-0">
-        <h1 className="font-bold text-4xl text-center text-col-secondary text-gray-800">Experience the <span className="text-green-500">benefits</span> of ZappNode.</h1>
+        <h1 className="font-bold text-4xl text-center color-gray">Experience the <span className="text-green-500">benefits</span> of ZappNode.</h1>
```

Comment:
The dark grey text color and light grey background color are still failing the contrast checker test: https://webaim.org/resources/contrastchecker/?fcolor=777777&bcolor=F3F3F3